app/models.py

```diff
@@ -696,7 +696,7 @@ def remove_domain(self, name):
             db.session.delete(sd)
             db.session.flush()
 
-    def update_domain_assessment_status(self, name_or_id, status, user=''):
+    def update_domain_assessment_status(self, name_or_id, status, user='', data={}):
```

Comment:
`data` should not be mutable: https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html

May need to be `data=None` and early in the function it is assigned to an empty dict if it isn't already one.

Reply:
I've updated `user` too.  Thanks for the link.  That will be a good read! 👍